src/components/dashboard/main-content/course-enrollments/tests/CourseEnrollments.test.jsx

```diff
@@ -60,28 +59,53 @@ function renderEnrollmentsComponent(initialProps) {
     </Provider>,
   );
 }
+describe('Course enrollements', () => {
+  beforeEach(() => {
+    updateCourseCompleteStatusRequest.mockImplementation(() => {});
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  test('loads enrollments component', () => {
+    renderEnrollmentsComponent(initProps);
+    expect(screen.getByText('My courses in progress')).toBeInTheDocument();
+    expect(screen.getByText('Completed courses')).toBeInTheDocument();
+    expect(screen.getByText('Courses saved for later')).toBeInTheDocument();
+    expect(screen.getAllByText('edX Demonstration Course').length).toBeGreaterThanOrEqual(1);
+  });
 
-test('loads enrollments component', () => {
-  renderEnrollmentsComponent(initProps);
-  expect(screen.getByText('My courses in progress')).toBeInTheDocument();
-  expect(screen.getByText('Courses saved for later')).toBeInTheDocument();
-  expect(screen.getAllByText('edX Demonstration Course').length).toBeGreaterThanOrEqual(1);
-});
+  test('move to in progress action generates course status update', () => {
+    renderEnrollmentsComponent({
+      ...initProps,
+      courseRuns: {
+        ...initProps.courseRuns,
+      },
+    });
+    expect(screen.getByRole('button', { name: 'Move course to In Progress' })).toBeInTheDocument();
+    act(() => {
```

Comment:
The RTL docs say we should not need act() for fireEvent.. also mentioned here: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#wrapping-things-in-act-unnecessarily did you run into warnings without it?

Reply:
(discussed on slack, adding here for posterity)
I am getting warnings about it, but they don't seem to be going away since I added it, so I'm going to remove it.